news/2021-09-01-new-featured-artist-alkome.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "New Featured Artist: Alkome"
+date: 2021-09-01 09:00:00 +0000
+---
+
+We're proud to welcome **Alkome** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/195/header.jpg)
+
+Releasing music since 2018 with expertise of a veteran, **Alkome** is a Japanese producer who keen eyes may recognize from [*Alkome - Mind Disorder*](https://osu.ppy.sh/beatmapsets/1301375) in the [OWC 2020](https://osu.ppy.sh/wiki/en/Tournaments/OWC/2020) semifinals map pool. Known for his mesh of concepts from the UK Hardcore genre and other styles across the spectrum, **Alkome**'s modern approach to music has already been adopted by rhythm games like *SOUND VOLTEX* and *SEVEN's CODE*.
```

Comment:
```suggestion
Releasing music since 2018 with the expertise of a veteran, **Alkome** is a Japanese producer whom keen eyes may recognize from [*Alkome - Mind Disorder*](https://osu.ppy.sh/beatmapsets/1301375) in the [OWC 2020](https://osu.ppy.sh/wiki/en/Tournaments/OWC/2020) semifinals map pool. Known for his mesh of concepts from the UK Hardcore genre and other styles across the spectrum, **Alkome**'s modern approach to music has already been adopted by rhythm games like *SOUND VOLTEX* and *SEVEN's CODE*.
```